pkg/controller/atlasdeployment/backup.go

```diff
@@ -26,6 +27,10 @@ import (
 	mdbv1 "github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/v1"
 )
 
+var errArgIsNotBackupSchedule = errors.New("failed to match resource type as AtlasBackupSchedule")
+
+const BackupProtected = "unable to reconcile AtlasBackupSchedule: it already exists in Atlas, it was not previously managed by the operator, and the deletion protection is enabled"
```

Comment:
The error message is standardized among the resources and must point to the docs
```unable to reconcile RESOURCE_NAME due to deletion protection being enabled. see https://dochub.mongodb.org/core/ako-deletion-protection for further information```

Reply:
Thanks! Fixed